scripts/check.sh

```diff
@@ -27,4 +27,36 @@ case $1 in
             echo 'Please re-generate schema with `cargo run --release --bin kagami -- schema > docs/source/references/schema.json`'
             exit 1
         };;
+    "docker-compose")
+        do_check() {
+            cmd_base=$1
+            target=$2
+            # FIXME: not nice; add an option to `kagami swarm` to print content into stdout?
+            #        it is not a default behaviour because Kagami resolves `build` path relative
+            #        to the output file location
+            temp_file="docker-compose.TMP.yml"
```

Comment:
Is there a better way?

Reply:
I think on linux it is possible to use `/dev/stdout` to hack your way into a program not supporting it